content/en/docs/setup/install/operator/index.md

```diff
@@ -75,11 +73,197 @@ accessible to `istioctl` by using this command:
 $ istioctl experimental profile list
 {{< /text >}}
 
-Optionally, you can use the `-s` flag with a install package path to see the
-list of configuration profiles available for other Istio versions:
+## Customize Istio settings using the IstioControlPlane API
+
+You can change a feature or component setting by using the [IstioControlPlane API](https://github.com/istio/operator/blob/release-1.3/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto).
+
+### Identify the feature or component settings
+
+The API groups Istio control plane components by feature, as shown in the table below: 
+
+| Feature | Components |
+|---------|------------|
+Base | CRDs
+Traffic Management | Pilot
+Policy | Policy
+Telemetry | Telemetry
+Security | Citadel
+Security | Node agent
+Security | Cert manager
+Configuration management | Galley
+Gateways | Ingress gateway
+Gateways | Egress gateway
+AutoInjection | Sidecar injector
+
+In addition to the core Istio components, third-party addon features and components are also available:
+
+| Feature | Components |
+|---------|------------|
+Telemetry | Prometheus
+Telemetry | Prometheus Operator
+Telemetry | Grafana
+Telemetry | Kiali
+Telemetry | Tracing
+ThirdParty | CNI
+
+Features can be enabled or disabled, which enables or disables all of the components that are a part of the feature.
+Namespaces that components are installed into can be set at the global, feature, or component level, in increasing
+order of override priority.
+
+### Configure the feature or component settings
+
+After you identify the name of the feature or component from the previous table, you can use the API to set the values
+using the `--set` flag, or create an overlay file and pass it with the `--filename` flag. In general, the `--set` flag
```

Comment:
I'd remove In general, or you wind up in the trap of operators asking "what is not the .general chase?  I understand why this is the case as a developer, but I struggle to understand where this logic _wouldn't_ apply in all cases.

This also seems like a long sentence to chomp down.  Periods are your frfends when people are reading documenation 👍

Reply:
Done.